ghost/admin/app/components/gh-post-settings-menu.js

```diff
@@ -604,4 +604,16 @@ export default class GhPostSettingsMenu extends Component {
     setSidebarWidthVariable(width) {
         document.documentElement.style.setProperty('--editor-sidebar-width', `${width}px`);
     }
+
+    get canViewPostHistory() {
+        let showPostHistory = this.feature.postHistory === true
+            && this.post.lexical
+            && this.post.emailOnly === false;
+
+        if (this.post.isPublished === true) {
+            showPostHistory = this.post.hasEmail === false;
+        }
```

Comment:
Similar code suggestion as earlier
```suggestion
        if (this.post.isPublished) {
             showPostHistory = !this.post.hasEmail
        }
```